packages/electrode-redux-router-engine/lib/redux-router-engine.js

```diff
@@ -56,7 +56,11 @@ class ReduxRouterEngine {
             `redux-router-engine: ${location} doesn't allow request method ${req.method}`);
         }
 
-        return this._handleRender(req, match, options || {});
+        try {
+          return this._handleRender(req, match, options || {})
+        } catch(err) {
```

Comment:
there is an error in the index-view.js's `createReduxStore` promise. so the promise never gets resolved and hence never ends up the catch block of the promise chain.